free-1.md

```diff
@@ -4,65 +4,176 @@
 This section describes who the project would serve and why a language might be a
 good way to meet their needs.
 
-
 ### What's the need?
 _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+This project is designed to serve individuals who need to schedule meetings
+among a large group.
+
+Currently the de-facto standards for scheduling are one
+of the following two methods:
+ * Everyone send out their availability for a certain time and a leader
+picks the times that seem to work best for a plurality.
+ * Everyone uses some software, such as a WhenIsGood&#8482; or When2Meet&#8482;
+and choses a binary yes/no for each time slot.
+
+If I was able to help them, users would be able to specify, in a way that feels
+natural, their availability and preferences.  Hopefully this can reduce either
+the work of an individual (who must compile the results of an email) or the
+repetition inherent in painting over the same boxes for pairs of weeks when
+using a WhenIsGood&#8482;.
```

Comment:
Interesting pretty universally applciable idea.  I definitely have run into this problem trying to schedule things with whenisgood and finding that no time works for everyone but clearly some people could meet at times they didn't paint over for some reason.  Probably they painted the times that were best for them but not the times that also worked for them...